common/cgroup/cgroup.go

```diff
@@ -252,6 +253,37 @@ func CreateCgroups(root string, subcgroup string, serviceNames []string) error {
 		// directories to mount read-write
 		subcgroupPath := filepath.Join(cPath, subcgroup)
 
+		// Workaround for https://github.com/coreos/rkt/issues/1210
+		// If /machine.slice does not exist, create it and configure it
+		cgroupPathFix := filepath.Join(cPath, "system.slice")
+		if err := os.MkdirAll(cgroupPathFix, 0755); err != nil {
```

Comment:
Here we always create `/sys/fs/cgroup/<CONTROLLER>/system.slice`. Maybe we should just check if the controller is `cpuset` and only then do the workaround.